csrc/ir_nodes.cpp

```diff
@@ -1603,13 +1603,13 @@ std::string GatherOp::toInlineString(int indent_size) const {
   TORCH_CHECK(false, "Tensor op can not be printed inline");
 }
 
-int GatherOp::gatherAxis(int axis) const {
+size_t GatherOp::gatherAxis(size_t axis) const {
```

Comment:
The parameter is intended to be an signed integer so that negative indexing is allowed like `TensorVie::axis(int)`. 

Does the parameter type need to be changed? Our usage of integer types seems rather inconsistent. I think here I intended it to follow the same pattern as `TensorView::axis(int)`. Using `int` however results in lots of type casting. Perhaps we should stick to `int64_t`?

Reply:
Thanks. Can you change these to `int64_t`? That seems like the most sensible design.